### ISSUE TITLE
Request for Hyperparameters to Reproduce Full Fine-Tuning Results in GLUE

### ISSUE THREAD

**Issue by HarryWu99** [QingruZhang/AdaLoRA#1]
Thanks for your fantastic work!

Now I want to reproduce full Fine-tuning results in Glue using DeBERTaV3-base, but it seems that no specific hyperparameters are given in the paper or in this repo. Specifically, I would like to know the following hyperparameters:

- learning rate
- weight_decay
- random seed

Thanks for your help!

**Comment by QingruZhang** [comment]
Hello Guanyu, 
Thanks for your interests in our paper. You can find the specific runing commands to reproduce the NLU results in [this folder](https://github.com/QingruZhang/AdaLoRA/tree/main/NLU/scripts). Our paper has listed the hyperpapermeters in Appendix C. Please let me know if you get any questions when running these commands or if you are reproducing the results of different target rank. I will post you the specific configs. 
Thanks,
Qingru
